index.html

```diff
@@ -147,8 +176,9 @@ <h5 class="card-heading heading-color">Skills</h5>
 </main>
 
 <footer class="footing background">
+    <div class="footer-main-container">
 <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
-<div>
+<div class="footer-right-container">
 <form class="form-container user-form" action="https://formspree.io/f/xbjqyrpj" method="POST">  
     <input type="text" class="user-info name" placeholder="Full name" name="user-name" maxlength="30" required>
     <input type="email" class="user-info" placeholder="Email address" name="email-address" required>
```

Comment:
- [ ] Move the "get in touch" button from the center to the left

Reply:
I have done it, thank you